scripts/enrich_event_feed.py

```
@@ -0,0 +1,464 @@
+    if notes:
+        result["eventNotes"] = notes
+    if not any(key in result for key in DETAIL_FIELDS) and intro:
+        result["eventNotes"] = intro
+    return result
+
+
+def fetch_html(url: str, timeout: int = 15, attempts: int = 3, backoff_seconds: float = 0.75) -> str:
+    request = urllib.request.Request(
+        url,
+        headers={"User-Agent": USER_AGENT, "Accept": "text/html,application/xhtml+xml"},
+    )
```

Ruff (0.16.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)